### ISSUE TITLE
CORS issues still present?

### ISSUE THREAD

**Issue by thilojaeggi** [devgianlu/go-librespot#119]
So I just cloned the latest version and tried to connect to the websocket in my flutter app but keep getting a connection refused error.
This is my config:
```
device_name: nordan
log_level: debug
credentials:
  type: zeroconf
device_type: speaker
server:
  enabled: true
  address: ""
  port: 3678
  allow_origin: "*"
  ```

**Comment by thilojaeggi** [comment]
Weird, with the pre built binary for [v0.0.17.1](https://github.com/devgianlu/go-librespot/releases/tag/v0.0.17.1) it works?

**Comment by aykevl** [comment]
What does your log say?
It should be something like the following:

```
INFO[0000] api server listening on [::]:3678       
```

**Comment by devgianlu** [comment]
@thilojaeggi Please check that the configuration file is being read properly, you can find how to migrate in #103:

> The simplest option to keep the previous behavior is by using `--config_dir=.`. This will keep reading the configuration from the current working directory, and stores the app state there.
To move the configuration to the new location, use:
> ```
> $ mkdir ~/.config/go-librespot
> $ mv credentials.json config.yml ~/.config/go-librespot/
> ```

@aykevl I think the README should be updated detailing how the configuration works after #103.

**Comment by devgianlu** [comment]
@thilojaeggi I have updateded the README to have a more detailed explaination on how the configuration works.

I'll be closing this as `connection refused` is most definitely not a CORS issue. Feel free to open another issue if needed.

**Comment by aykevl** [comment]
> @aykevl I think the README should be updated detailing how the configuration works after #103.

Oops, sorry I missed the other `server:` configuration in the README! Thanks for fixing!
